Classes/Xclasses/LocalizationController.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare(strict_types=1);
+
+namespace B13\Container\Xclasses;
+
+/*
+ * This file is part of TYPO3 CMS-based extension "container" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+use B13\Container\Tca\Registry;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Http\JsonResponse;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class LocalizationController extends \TYPO3\CMS\Backend\Controller\Page\LocalizationController
+{
+    /**
+     * @var Registry
+     */
+    protected $containerRegistry;
+
+    public function __construct(Registry $containerRegistry = null)
+    {
+        parent::__construct();
+        $this->containerRegistry = $containerRegistry ?? GeneralUtility::makeInstance(Registry::class);
+    }
+
+    public function getRecordLocalizeSummary(ServerRequestInterface $request): ResponseInterface
+    {
+        $response = parent::getRecordLocalizeSummary($request);
+        $payload = json_decode($response->getBody()->getContents(), true);
+        $payload = $this->rebuildPayload($payload);
+        return new JsonResponse($payload);
+    }
```

Comment:
Maybe keep everything but this method out if the XClass so it does not seem so bloated.